runtime/queries/capnp/highlights.scm

```diff
@@ -0,0 +1,141 @@
+; Preproc
+
+(unique_id) @preproc
```

Comment:
It looks like these queries use neovim captures. We use a slightly different set of captures documented here: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

For example, this should be `@keyword.directive`, `@include` should be `@keyword.control.include`, and `@method` should be `@function.method`

Reply:
Updated in [ac50eb3](https://github.com/helix-editor/helix/pull/6325/commits/ac50eb3fd5ead297a5ef5c277f99b4057ed927be)

Reply:
Thanks for the review! The result is much better with the correct set of captures.